cmd/drand-cli/cli.go

```diff
@@ -1168,6 +1184,17 @@ func contextToConfig(c *cli.Context, l log.Logger) *core.Config {
 		opts = append(opts, core.WithTrustedCerts(paths...))
 	}
 
+	if c.IsSet(tracesFlag.Name) {
+		opts = append(opts, core.WithTracesEndpoint(c.String(tracesFlag.Name)))
+	}
+
+	if c.IsSet(tracesProbabilityFlag.Name) {
+		opts = append(opts, core.WithTracesProbability(c.Float64(tracesProbabilityFlag.Name)))
+	} else {
+		//nolint:gomnd // Reset the trace probability to 5%
+		opts = append(opts, core.WithTracesProbability(0.05))
```

Comment:
is it possible to use the default value if the flag is not set? Just in case a mismatch happens between the flag default value and the value set here

Reply:
I meant rather that we set a default value in the `CliFlag` and then repeat that value here, rather than referencing the one from the Flag